simplq/src/components/pages/Admin/AddMember.jsx

```diff
@@ -17,6 +17,7 @@ export default (props) => {
           buttonName="Add"
```

Comment:
Can we solution this around `buttonName` rather than going this route?

Reply:
Okay

Reply:
coz I think that will be a safer solution.